src/lib/shared/utilities.ts

```diff
@@ -1,8 +1,10 @@
-const toFile = ({ bits, name, lastModified, type }: Data) => ({
+import { DEFAULT_SIP_CONFIG } from '$lib/constants';
+
+export const toBase64 = ({ bits, name, lastModified, type }: Data) => ({
   bits: 'data:' + type + ';base64,' + bits.toString('base64'),
   name,
   lastModified,
-  type: type
+  type
 });
```

Comment:
The `toBase64` name is misleading since this is not a string, but an object corresponding to the fields of [File](https://developer.mozilla.org/en-US/docs/Web/API/File/File).